### ISSUE TITLE
Midterm Peer Review

### ISSUE THREAD

**Issue by meredithdobrzynski** [GraceJiangJinyi/ORIE4741_Project#8]
3 things I liked:
- In the final section, it is mentioned that data from November-December will be used to test the models again and analyze the result. While typically random sampling is used to create test set, there is an important temporal aspect to the data, so I liked this approach better. 
- The schedule included at the end of the midterm report was a good idea. Not only does it help to keep the group members on track, but it also gives me something to expect in the final report. They included that they would test their models and adjust for overfitting or underfitting, which is a crucial step in building predictive models. It is mentioned both cross-validation and regularization will be used to make these adjustments. 
- In features and transformations, it is stated that the data is standardized for the neural network model. This is not something I originally considered in my own project, but I will definitely be considering it now. Normalizing the data will help lead to faster convergence in the model which is very helpful for training. 

3 things to improve: 
- The information included about the features was similar to what was included in the proposal. I would have liked to see some more specific information about each feature in addition to the graphs included. Some examples would include data type (real valued, boolean, categorical, etc) and summaries about each column such as mean, min, max for real valued columns. 
- It is mentioned that SVR and neural networks are going to be constructed in the upcoming weeks, so I assume that the linear models have already been built. It would have been nice to see the results of the linear regression model including error, accuracy and a graph. 
- While the group had not begun constructing the neural network, it would have been nice to include some initial thoughts on how to construct the model such as which hyper-parameters to tune and what values to test for these hyper-parameters. Examples include: learning rate, number of epochs, number of hidden layers, etc.
